history/characters/1_azerothian.txt

```diff
@@ -61,8 +72,14 @@
 	martial=6 diplomacy=7 intrigue=5 stewardship=7 learning=4
 	trait=education_diplomacy_3 trait=chaste trait=patient
 	disallow_random_traits = yes
-	502.6.9 = { birth=yes }
-	587.1.1 = { death=yes }
+	496.6.9 = { 
+		birth=yes trait=creature_human 
+		effect = { make_important_lore_character_effect = yes } 
+	}
+	507.7.16 = {
+		add_spouse = 1	#Barathen
```

Comment:
Also I think Varia should be heterosexual as well if Barathen is

Reply:
fixed